### PR TITLE
PDF embedding

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
The markdown rendering component

### Description
This PR adds embedding of pdf links via `{%pdf LINK %}`

### Steps

- [x] added implementation
- [ ] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### CODE REVIEW

File: src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { RegexOptions } from '../../../../external-types/markdown-it-regex/interface'
+
+export const replacePdfShortCode: RegexOptions = {
+  name: 'legacy-youtube-short-code',
```
Comment (ErikMichelson) [inline src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts:4]:
```suggestion
  name: 'legacy-pdf-short-code',
```

File: src/components/editor/markdown-preview/replace-components/pdf/pdf-frame.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { DomElement } from 'domhandler'
+import React, { ReactElement } from 'react'
+import { OneClickEmbedding } from '../one-click-frame/one-click-embedding'
+import { getIdFromCodiMdTag, VideoFrameProps } from '../video-util'
+import './pdf-frame.scss'
+
+const getElementReplacement = (node: DomElement, counterMap: Map<string, number>): (ReactElement | undefined) => {
+  const videoId = getIdFromCodiMdTag(node, 'pdf')
```
Comment (ErikMichelson) [inline src/components/editor/markdown-preview/replace-components/pdf/pdf-frame.tsx:8]: Just a minor thing: Rename `videoId` to `pdfUrl` or similar?

File: src/components/editor/markdown-preview/replace-components/pdf/pdf-frame.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  if (videoId) {
+    const count = (counterMap.get(videoId) || 0) + 1
+    counterMap.set(videoId, count)
+    return <PdfFrame key={`youtube_${videoId}_${count}`} id={videoId}/>
```
Comment (ErikMichelson) [inline src/components/editor/markdown-preview/replace-components/pdf/pdf-frame.tsx:12]:
```suggestion
    return <PdfFrame key={`pdf_${videoId}_${count}`} id={videoId}/>
```

**Comment by ErikMichelson** [comment]
I just fixed the issues that I found.

File: src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { RegexOptions } from '../../../../external-types/markdown-it-regex/interface'
+
+export const replacePdfShortCode: RegexOptions = {
+  name: 'legacy-pdf-short-code',
```
Comment (DerMolly) [inline src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts:4]: shold it really be called legacy if there is no new version?

Comment (mrdrogdrog) [reply src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts:4]: fixed

**Pull request merged**